_data/simple-icons.json

```diff
@@ -5190,6 +5190,11 @@
             "hex": "0500BE",
             "source": "https://okcupid.com/press"
         },
+        {
+            "title": "Okta",
+            "hex": "338CCC",
+            "source": "https://www.okta.com/press-room/media-assets/auras/"
```

Comment:
Given that the SVG used is on the homepage, I think we'd be best to link there as the source. While the 'auras' show the treatment we give it, that page in particular doesn't include the SVG used. The hex is also `#007DC1` when pulled straight from the SVG, so maybe we should use that as our colour?

Reply:
I think https://www.okta.com/press-room/media-assets/ would suffice for the source as it includes the SVG logo in the header, confirms our treatment of it and may be the best source for future contributors to find changes to the treatment of the colour.

Reply:
Updated